.github/workflows/publish.yml

```
@@ -0,0 +1,261 @@
+name: Publish Docker image, Helm chart and Wire build
+
+on:
+  push:
+    branches: [master, dev, edge, avs, mobile, acc, stefan]
+    tags:
+      - '*staging*'
+      - '*production*'
+
+concurrency:
+  group: ci-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    name: Build Docker image and Helm Chart
+    runs-on: buildjet-8vcpu-ubuntu-2204
+
+    outputs:
+      wire_builds_target_branches: ${{ steps.output_target_branches.outputs.wire_builds_target_branches }}
+      image_tag: ${{ steps.push_docker_image.outputs.image_tag }}
+      release_name: ${{ steps.push_docker_image.outputs.release_name }}
+      chart_version: ${{ steps.publish_helm_chart.outputs.chart_version }}
+
+    env:
+      TEST_COVERAGE_FAIL_THRESHOLD: 45
+      TEST_COVERAGE_WARNING_THRESHOLD: 60
+      COMMIT_URL: ${{github.event.head_commit.url}}
+      COMMITTER: ${{github.event.head_commit.committer.name}}
+
+    steps:
+      - uses: azure/setup-helm@v3
+        with:
+          version: '3.12.2'
+        id: install
+
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 16.x
+          cache: 'yarn'
+
+      - name: Set environment variables
+        run: |
+          echo "BRANCH_NAME=$(git branch --show-current)" >> $GITHUB_ENV
+          echo "TAG=$(git tag --points-at ${{github.sha}})" >> $GITHUB_ENV
+          echo "PR_LAST_COMMIT_MESSAGE=$(git log --format=%B -n 1 ${{github.event.after}} | head -n 1)" >> $GITHUB_ENV
+
+      - uses: kanga333/variable-mapper@master
+        name: Determine target branches in wireapp/wire-builds
+        with:
+          key: '${{env.BRANCH_NAME}}'
+          map: |
+            {
+              "stefan": { "wire_builds_target_branches" : "[\"main\"]" },
+              "dev": { "wire_builds_target_branches" : "[\"main\"]" }
+            }
+
+      - name: Make target branches available as output to other jobs
+        id: output_target_branches
+        shell: bash
+        run: |
+          echo "wire_builds_target_branches: $wire_builds_target_branches"
+          echo "wire_builds_target_branches=$wire_builds_target_branches" >> $GITHUB_OUTPUT
+
+      - name: Set TITLE and BUILD_DESKTOP
+        env:
+          PR_TITLE: ${{github.event.pull_request.title || env.PR_LAST_COMMIT_MESSAGE}}
+        run: |
+          echo "TITLE=${PR_TITLE}" >> $GITHUB_ENV
+          echo "BUILD_DESKTOP=${{contains(env.TAG, 'staging') || contains(env.TAG, 'production') || contains(env.PR_LAST_COMMIT_MESSAGE, '+Desktop')}}" >> $GITHUB_ENV
```

```suggestion
      - name: Set TITLE 
        env:
          PR_TITLE: ${{github.event.pull_request.title || env.PR_LAST_COMMIT_MESSAGE}}
        run: echo "TITLE=${PR_TITLE}" >> $GITHUB_ENV
```
You don't need this desktop build here :)

---

fixed in 825fa975c08368600f09b615239f3b0a6f9a64d2